jwplayer-appletv-web-app/js/views/ListCollection.js

```diff
@@ -20,9 +20,12 @@ ViewManager.registerView("ListCollection", function(doc) {
   var listLoader = new PlaylistLoader();
   var collectionDoc = doc;
 
-  var playlists = collectionDoc.firstChild.getAttribute("data-playlists").split(",");
-  var featured = collectionDoc.firstChild.getAttribute("data-featured");
-  if (featured === "undefined") featured = undefined;
+  // Unfortunately we can not read this from TVML anymore since JSONArrays
+  // and Objects don't serialize well with JS template syntax.
+  // For this reason we read this from the global scope.
+  // Unfortunately this also makes this template less flexible.
+  var playlists = CONFIG.playlists;
```

Comment:
It's not necessary to serialize the entire playlist object, since a global hash of all playlists is already stored (`PLAYLISTS`).  That being said, if the `autoAdvance` settings are made global, this is no longer an issue.